src/components/accordion/accordion.spec.tsx

```diff
@@ -61,18 +65,21 @@ const isCollapsed = (wrapper) => {
     },
     wrapper.find(StyledAccordionContentContainer)
   );
-};
+}
 
 testStyledSystemSpacing(
-  (props) => <Accordion title="foo" {...props} />,
+  (props: AccordionProps) => <Accordion {...props} title="foo" />,
   {},
-  (wrapper) => wrapper.find(StyledAccordionContainer)
+  (wrapper: ReactWrapper) => wrapper.find(StyledAccordionContainer)
 );
 
 describe("Accordion", () => {
-  let wrapper;
+  let wrapper: ReactWrapper;
 
-  const render = (props) => {
+  const render = (
+    props?: Pick<Partial<AccordionProps>, "title"> &
```

Comment:
in this case it would (as there is only one required prop), but I prefer it that way, as if another required prop would be introduced to the Accordion component, we'll clearly see that it's missing in these tests.